modules/standard/Curl.chpl

```diff
@@ -446,341 +446,341 @@ proc slist.free() {
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_ADDRESS_SCOPE.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_ADDRESS_SCOPE.html */
 extern const curlopt_address_scope              : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_CERTINFO.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_CERTINFO.html */
 extern const curlopt_certinfo                   : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_USERNAME.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_USERNAME.html */
 extern const curlopt_username                   : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_PASSWORD.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_PASSWORD.html */
 extern const curlopt_password                   : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_PROXYUSERNAME.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_PROXYUSERNAME.html */
 extern const curlopt_proxyusername              : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_PROXYPASSWORD.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_PROXYPASSWORD.html */
 extern const curlopt_proxypassword              : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_NOPROXY.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_NOPROXY.html */
 extern const curlopt_noproxy                    : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_TFTP_BLKSIZE.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_TFTP_BLKSIZE.html */
 extern const curlopt_tftp_blksize               : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_SOCKS5_GSSAPI_SERVICE.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_SOCKS5_GSSAPI_SERVICE.html */
 extern const curlopt_socks5_gssapi_service      : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_SOCKS5_GSSAPI_NEC.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_SOCKS5_GSSAPI_NEC.html */
 extern const curlopt_socks5_gssapi_nec          : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_PROTOCOLS.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_PROTOCOLS.html */
 extern const curlopt_protocols                  : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_REDIR_PROTOCOLS.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_REDIR_PROTOCOLS.html */
 extern const curlopt_redir_protocols            : c_int ;
 /* All other curlopt values will be less than this one */
 extern const curlopt_lastentry                  : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_MAIL_FROM.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_MAIL_FROM.html */
 extern const curlopt_mail_from                  : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_MAIL_RCPT.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_MAIL_RCPT.html */
 extern const curlopt_mail_rcpt                  : c_int ;
-/* See http://curl.haxx.se/libcurl/c/CURLOPT_MAIL_AUTH.html */
+/* See https://curl.haxx.se/libcurl/c/CURLOPT_MAIL_AUTH.html */
```

Comment:
Daunting task, but have you verified that all these links are valid?  I know there are some sites that don't actually provide an https version, so this auto update would break there (right?)